app/assets/javascripts/request_service.js

```diff
@@ -0,0 +1,34 @@
+var RequestService = function() {
```

Comment:
If you think about it, using a constructor:

1. Adds the memory overhead of needing a constructor function, an instance, and a prototype object
1. Adds unnecessary complexity by having all of the things in the first one.

Really, in this case it's just serving as a namespace for a few methods. We don't need all that mental overhead.